components/apimgt/org.wso2.carbon.apimgt.keymgt/src/main/java/org/wso2/carbon/apimgt/keymgt/token/APIMJWTGenerator.java

```diff
@@ -138,7 +157,7 @@ public String buildBody(JwtTokenInfoDTO jwtTokenInfoDTO) throws APIManagementExc
 
         //generating expiring timestamp
         long currentTime = System.currentTimeMillis();
-        long expireIn = currentTime + getTTL() * 1000;
+        long expireIn = currentTime + getTTL() * 1000 + jwtTokenInfoDTO.getExpirationTime() * 1000;
```

Comment:
shall we put a comment explaining what there 2 parts of this? getTTL() and getExpirationTime()

Reply:
removed the getTTL method since it gets a time ti live for a JWT from the config and its not required in our use case. thanks for pointing out.